source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -19,14 +26,16 @@ real-time changes to your database. A change stream is a {+mdb-server+}
 feature that allows your application to subscribe to data changes on a single
 collection, database, or deployment. You can specify a set of aggregation
 operators to filter and transform the data your application receives. When
-connecting to a MongoDB deployment v6.0 or later, you can configure the
-events to include the document data before and after the change.
+connecting to a MongoDB deployment running version 6.0 or later, you can
+configure the events to include the document data before and after the
+change.
```

Comment:
👍 
Alternatively, "When connecting to MongoDB v6.0 or later, ..." but I guess this version makes it consistent to the intro paragraph of Split Large Change Stream Events.

Reply:
changed for concision!